run-tests.php

```diff
@@ -1567,10 +1561,6 @@ function run_test($php, $file, $env)
 					show_result('SKIP', $tested, $tested_file, '', $temp_filenames);
 				}
 
-				if (isset($old_php)) {
-					$php = $old_php;
-				}
-
```

Comment:
I've tracked such occurrences carefully and it feels absolutely safe to me.
However, it you find this is not true - let me know with example. This is a hardcore spaghetti code, so probability that I'm wrong is higher than 0.